testworld/document_consensus_test.go

```diff
@@ -12,7 +13,6 @@ import (
 )
 
 func TestHost_AddExternalCollaborator(t *testing.T) {
-	t.Parallel()
```

Comment:
Is there any main reason to making this test serial?

Reply:
When running this locally, the test failed because of this.